unity_package/Assets/SteamVR/Editor/SteamVR_RenderModelEditor.cs

```diff
@@ -86,7 +86,7 @@ public override void OnInspectorGUI()
 
 		GUILayout.BeginHorizontal();
 		GUILayout.Label("Model Override");
-		var selected = EditorGUILayout.Popup(renderModelIndex, renderModelNames);
+		var selected = EditorGUILayout.Popup(renderModelIndex, renderModelNames ?? new string[0]);
```

Comment:
It has (probably) problem when 2nd parameter (renderModelNames) being null.

`renderModelNames ?? new string[0]`

is equivalent to 

`renderModelNames == null ? renderModelNames : new string[0]`